src/telemetry/index.ts

```diff
@@ -96,16 +96,29 @@ export class TelemetryService implements vscode.Disposable {
           .get<boolean>("isLocalMode", false)
           ? "true"
           : "false",
-        stack:
+        stack: this.removeGenericSecretsFromStackTrace(
           error !== undefined && error instanceof Error
             ? error.stack
             : JSON.stringify(error),
+        ),
         ...this.customAttributes,
       },
       measurements,
     );
   }
 
+  private removeGenericSecretsFromStackTrace(
```

Comment:
instead of removing, should we not redact and replace with "****" or some other marker so that we know secret was redacted